cate/ops/coregistration.py

```diff
@@ -309,6 +314,11 @@ def _resample_dataset(ds_master: xr.Dataset, ds_slave: xr.Dataset, method_us: in
     lat = ds_master['lat'].sel(lat=lat_slice)
     ds_slave = ds_slave.sel(lon=lon_slice, lat=lat_slice)
 
+    if np.array_equal(lon.values, ds_slave['lon'].values) and \
+       np.array_equal(lat.values, ds_slave['lat'].values):
```

Comment:
Can we put an epsilon, e.g. 1e-8 here?

Reply:
Yes, let's use:
https://docs.scipy.org/doc/numpy-1.15.1/reference/generated/numpy.allclose.html